### PR TITLE
upgrade rspec

### PR DESCRIPTION
**Pull request by lihanli**
upgrade rspec to fix this deprecation
```
[DEPRECATION] `last_comment` is deprecated.  Please use `last_description` instead.
```

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -30,9 +30,8 @@ group :development, :test do
   gem 'bundler-audit'
 
   # Testing tools
-  gem 'rspec'
-  gem 'rspec-rails'
-  gem 'guard-rspec'
+  gem 'rspec-rails', '3.5.1'
```
Comment (markolson) [inline Gemfile]: Can we loosen these requirements to just major version?

**Comment by lihanli** [comment]
@markolson ok i changed it

**Comment by saneshark** [comment]
@lihanli please upmerge

**Comment by saneshark** [comment]
LGTM

**Pull request merged**